src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -160,7 +160,14 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, en
     </Box>
   );
 
-  const pageTitle = `${wbsPipe(project.wbsNum)} - ${project.name}`;
+  let teamName = '';
+  try {
+    teamName = project.teams[0].teamName;
```

Comment:
Make sure you display all teams, not just the first one.

Reply:
Take a look at some examples in the codebase of how to construct a list from an array separated by commas